### DISCUSSION TITLE
About precision of printed numbers

### DISCUSSION THREAD

**Discussion by Spaghestti** [Estia-advanced-programming/pandora-public#80] [Q&A] [answered]
We wonder if we need to use doubles or floats (depending on the calculations) because some can be different at the 9th decimal. Or should we use only 2 decimals?

**Comment by dhmmasson** [comment, marked as answer]
Some elements of answers : 

- I don't know the point in this project of going for lower precision data type ( float ) **use ```double```**
- precision in the output should not exceed the precision in the input ( e.g. timestamp are in 100th of seconds ) 
- For this year let's keep it to maximum **two digits after the decimal point** 
```java
double value = 1.234 ; 
System.out.format("%.2f", value);
```

**Comment by WilliamDelamare** [comment]
Since you add directly string in the dictionary (map) 'featureValues', you can use:
- String.format("%.2f", variableName); 
instead of
- String.valueOf(variableName);
